pkg/catalog/manager/sync.go

```diff
@@ -61,20 +64,12 @@ func (m *Manager) Sync(key string, obj *v3.Catalog) error {
 		logrus.Debugf("Catalog %s is already up to date", catalog.Name)
 		if v3.CatalogConditionRefreshed.IsUnknown(catalog) {
 			v3.CatalogConditionRefreshed.True(catalog)
+			v3.CatalogConditionRefreshed.Reason(catalog, "")
```

Comment:
why are we setting Reason on a True condition?

Reply:
we have to offset the reason to empty in case it was set on other places.